Applications/web_portal/web_app/library/source/pages/dashboard_page/dashboard_page.tsx

```diff
@@ -39,22 +85,167 @@ export class DashboardPage extends React.Component<Properties, State> {
             </VBoxLayout>
           </HBoxLayout>
           <Padding/>
-          <VBoxLayout height='60px' width='45px'>
+          <VBoxLayout height='60px' width='45px' className={
+              css(DashboardPage.STYLE.notificationButtonWrapper)}>
             <Padding size='20px'/>
-            <NotificationButton items={0} isOpen={false}/>
+            <div style={{height: '20px', width: '15px'}}>
```

Comment:
Why is this div here? The dimensions of the button should be part of the NotificationButton, not the container div.